proxstar/tasks.py

```diff
@@ -227,8 +226,23 @@ def setup_template_task(template_id, name, user, ssh_key, cores, memory):
 
 
 def cleanup_vnc_task():
-    requests.post(
-        'https://{}/console/cleanup'.format(app.config['SERVER_NAME']),
-        data={'token': app.config['VNC_CLEANUP_TOKEN']},
-        verify=False,
-    )
+    """Removes all open VNC sessions. This runs in the RQ worker, and so
+    needs to be routed properly via the Proxstar API
+    TODO (willnilges): Use API, track the task IDs, and kill only the finished
+    ones every couple of minutes
+    https://github.com/ComputerScienceHouse/proxstar/issues/153
+    """
+    print('Clearing vnc targets')
```

Comment:
Debug print left

Reply:
This is another intentional log message because there's literally no other indication that this action was taken.